keyboards/sixkeyboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,33 @@
+
+#include QMK_KEYBOARD_H
+#include "matrix.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+    KC_A, KC_B, KC_C, \
+    KC_D, KC_E, KC_F  \
+  ),
+
+  [1] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  ),
+
+  [2] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  ),
+
+  [3] = LAYOUT(
+    KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS  \
+  )
+};
+
+void matrix_scan_user(void) {
+  // jump to bootloaer when all keys are pressed
+  if (matrix_get_row(0) == 0b111 && matrix_get_row(1) == 0b111) {
+    clear_keyboard();
+    bootloader_jump();
```

Comment:
This handles the clear keyboard and the like. 
```suggestion
    reset_keyboard();
```

https://github.com/qmk/qmk_firmware/blob/00fc35539d774c43cde383a8d2cc8b254787ce6c/quantum/quantum.c#L123-L145